setup.cfg

```diff
@@ -1,3 +1,33 @@
+[metadata]
+name = gsw
+description = Gibbs Seawater Oceanographic Package of TEOS-10
+author = Eric Firing, Filipe Fernandes
+author_email = efiring@hawaii.edu
+url = https://github.com/TEOS-10/GSW-python
+long_description = file: README.md
+long_description_content_type = text/markdown
+license = MIT
+license_file = LICENSE.txt
+classifiers =
+    Development Status :: 5 - Production/Stable
+    Intended Audience :: Science/Research
+    Operating System :: OS Independent
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Topic :: Scientific/Engineering
+
+
+[options]
+zip_safe = False
+install_requires =
+    numpy
+python_requires = >=3.6
```

Comment:
Is leaving this as '>=3.6' intentional, on the grounds that it should still work fine on 3.6?